packages/execution-environments/webpack.config.js

```diff
@@ -36,7 +37,35 @@ module.exports = (_, argv) => {
               'dist',
               'lockdown.umd.min.js',
             ),
-            to: path.resolve(DIST, 'lockdown.umd.min.js'),
+            to: path.resolve(ENVIRONMENTS, 'webworker/lockdown.umd.min.js'),
+            toType: 'file',
+          },
+        ],
+      }),
+      // @todo Deduplicate this
```

Comment:
This isn't just a matter of merging the two separate `patterns` arrays?

Reply:
I did that, but I think we can merge the two patterns as well, I just don't know how 😅